scripts/spawn-swarm.js

```diff
@@ -0,0 +1,12 @@
+const {swarmManager} = require('../src/swarmManager');
+require('../tests/test.configurations');
+
+(async () => {
+
+    const manager = await swarmManager();
+    const swarm = await manager.generateSwarm({numberOfDaemons: process.argv[2] || 3});
```

Comment:
Since we're pulling in the config anyways, perhaps we should use the existing `numOfNodes` to keep this consistent and centralized.

Reply:
defaulting to `numOfNodes`